### PR TITLE
Store topic queries in a csv file

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

**Comment by shifucun** [comment]
Updated csv files, this is ready to review.

File: server/config/home_page/topics.json [hunk 1]
```diff
@@ -1,55 +1,65 @@
-[{
-  "id": "agriculture",  
-  "title": "Agriculture",
+[
+  {
+    "id": "food",
+    "title": "Food",
     "description": "Discover the profound impact of climate change on agriculture centers worldwide",
```
Comment (pradh) [inline server/config/home_page/topics.json:5]: nit: food production and agriculture centers ?

File: server/config/home_page/topics.json [hunk 2]
```diff
@@ -1,55 +1,65 @@
...
+    "id": "sustainability",
+    "title": "Sustainability",
+    "description": "Embark on a journey towards a sustainable future.",
+    "url": "/nl#topic=sustainability",
+    "icon": "eco"
+  },
+  {
+    "id": "health",
+    "title": "Health",
+    "description": "Embark on a journey into the remarkable world of biomedical research and innovation.",
```
Comment (pradh) [inline server/config/home_page/topics.json:61]: TODO needs fixing?

File: static/js/apps/nl_interface/query_examples.tsx [hunk 3]
```diff
@@ -18,248 +18,47 @@
...
-  if (!topic || !TOPIC_QUERIES[topic]) {
+  if (!topic) {
     return null;
   }
-  const examples = TOPIC_QUERIES[topic];
+  const examples = [];
+  if (csvData) {
+    for (const row of csvData.slice(1)) {
+      if (row[0] === topic && row[2] === "yes") {
+        examples.push(row[1]);
```
Comment (pradh) [inline static/js/apps/nl_interface/query_examples.tsx:58]: Could you trim spaces?

File: static/js/apps/nl_interface/query_examples.tsx [hunk 4]
```diff
@@ -18,248 +18,47 @@
...
   const topic = getUrlToken("topic");
-  if (!topic || !TOPIC_QUERIES[topic]) {
+  if (!topic) {
     return null;
   }
-  const examples = TOPIC_QUERIES[topic];
+  const examples = [];
+  if (csvData) {
+    for (const row of csvData.slice(1)) {
+      if (row[0] === topic && row[2] === "yes") {
```
Comment (pradh) [inline static/js/apps/nl_interface/query_examples.tsx:57]: nit: Be case tolerant and maybe allow "y"?

lower(strip(row[1])) && (lower(strip(row[2])) === "yes" || lower(strip(row[2])) === "y")

**Review by pradh — APPROVED** [review]
Thanks!

**Review by chejennifer — APPROVED** [review]

File: static/js/apps/nl_interface/query_examples.tsx (hunk 4 above)
Comment (shifucun) [reply static/js/apps/nl_interface/query_examples.tsx]: Done

File: static/js/apps/nl_interface/query_examples.tsx (hunk 3 above)
Comment (shifucun) [reply static/js/apps/nl_interface/query_examples.tsx:59]: Done

File: server/config/home_page/topics.json [hunk 5]
```diff
@@ -58,7 +58,7 @@
   {
     "id": "health",
     "title": "Health",
-    "description": "Embark on a journey into the remarkable world of biomedical research and innovation.",
+    "description": "Explore various how health is affected by various social and bilogical factors",
```
Comment (pradh) [inline server/config/home_page/topics.json:61]: typo: biological.
also have a fullstop `.`

Comment (shifucun) [reply server/config/home_page/topics.json:61]: fixed

**Pull request merged**